yarn.lock

```diff
@@ -1224,32 +1224,32 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@jupyterlab/apputils@npm:^4.0.0":
-  version: 4.3.5
-  resolution: "@jupyterlab/apputils@npm:4.3.5"
+"@jupyterlab/apputils@npm:~4.0.0":
+  version: 4.0.0
```

Comment:
I'm surprised this get downgraded.

Reply:
So am I, it doesn't look right to me too.

Reply:
I think I shall try troubleshooting this issue for sometime locally first and then get back